### DISCUSSION TITLE
Sitting & Laying down system

### DISCUSSION THREAD

**Discussion by CapValGo** [EveronLife/EveronLife#128] [Ideas]
This is a general idea on being able to sit/lay down on chairs, couches, beds, park benches. All around the map.

Would this be something that is wanted to be done?

**Comment by Arkensor** [comment]
Yes! I would love the ability to do all these kinds of things, but I am unsure if this will be possible animation-wise. Right now it looks like we have a problem if we want to manipulate the player's animation graph. The only chance to add animations right now is via inventory items/weapons that can inject something into the graph. If we touch the main graph to add something like sitting down I think the framework becomes incompatible with other mods that want to add animations to the player. We will need to evaluate the options and think about maybe adding invisible fake items into the hands of players to make them appear to sit down.
